src/service/video-manager-impl.js

```diff
@@ -942,6 +967,11 @@ class VideoEntry {
     return PlayingStates.PLAYING_MANUAL;
   }
 
+  /** @return {boolean} */
+  isRollingAd() {
+    return this.isRollingAd_;
+  }
```

Comment:
getter?

Reply:
Will do later, keeping as-is for consistency with current getters.